### ISSUE TITLE
"Multiple entries with same key" error when trying to run instrumented tests for the release build

### ISSUE THREAD

**Issue by FrancoSabadini** [Instabug/Instabug-Android#197]
### Steps to Reproduce the Problem

Try to run instrumented tests for a **release** build of an app that uses Instabug.

These are the Gradle dependencies:

```
val instabugVersion = "8.2.2"
implementation("com.instabug.library:instabug:$instabugVersion")
implementation("com.instabug.library:instabug-with-okhttp-interceptor:$instabugVersion")
```

### Expected Behavior

Tests run as expected.

### Actual Behavior

Build fails with the following error:

```
> Task :app:transformClassesAndResourcesWithR8ForEnvDevReleaseAndroidTest FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:transformClassesAndResourcesWithR8ForEnvDevReleaseAndroidTest'.
> Multiple entries with same key: com.instabug.bug.view.e=com.instabug.bug.view.e(library class) and com.instabug.bug.view.e=com.instabug.bug.view.e(library class)
```

I tried removing the `instabug-with-okhttp-interceptor` dependency (and the respective code) and then the error change to the following:

```
> Task :app:transformClassesAndResourcesWithR8ForEnvDevReleaseAndroidTest FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:transformClassesAndResourcesWithR8ForEnvDevReleaseAndroidTest'.
> Multiple entries with same key: com.instabug.library.annotation.a.e=com.instabug.library.annotation.a.e(library class) and com.instabug.library.annotation.a.e=com.instabug.library.annotation.a.e(library class)
```

That is the same type of error but for a different class (notice that the path of the first error is `com.instabug.bug.view.e` and the second one is `com.instabug.library.annotation.a.e`).

### SDK Version

8.4.0

### Android Version

Android 9

### Device Model

Pixel 3

**Comment by FrancoSabadini** [comment]
After trying different things I made it work by adding the following to my `proguard-rules.pro` file:

```
-keep class com.instabug.**
```

This is not ideal though as it means none of the Instabug classes will be affected by R8. Maybe there are less restrictive rules we can use?

**Comment by amrbrkt** [comment]
Hi @FrancoSabadini, 
Thanks for reporting that, we'll be looking into it.
Would you please tell me if you are facing any kind of these problems if `R8` isn't enabled?

**Comment by FrancoSabadini** [comment]
I tried disabling R8, but then a bunch of other problems appeared (not
related to the Instabug SDK) and I didn't have time to fix them all so I
had to enable it back.

**Comment by mzelzoghbi** [comment]
Hey @FrancoSabadini , Can you provide us with some versions like gradle wrapper & build tools versions, support libraries version, robolectric / espresso / junit versions and finally android studio version. Thanks

**Comment by FrancoSabadini** [comment]
Hi @mzelzoghbi, here are the versions I was using when I saw this issue:

**Gradle wrapper version:** 5.1.1
**Android build tools:** 28.0.3
**AndroidX libs versions:** 
```
    implementation("androidx.fragment:fragment:1.0.0")
    implementation("androidx.annotation:annotation:1.0.2")
    implementation("androidx.appcompat:appcompat:1.0.2")
    implementation("androidx.cardview:cardview:1.0.0")
    implementation("androidx.recyclerview:recyclerview:1.0.0")
    implementation("androidx.constraintlayout:constraintlayout:1.1.3")

    val lifecycleVersion = "2.0.0"
    implementation("androidx.lifecycle:lifecycle-extensions:$lifecycleVersion")
    kapt("androidx.lifecycle:lifecycle-compiler:$lifecycleVersion")
    val roomVersion = "2.0.0"
    implementation("androidx.room:room-runtime:$roomVersion")
    kapt("androidx.room:room-compiler:$roomVersion")

    implementation("androidx.core:core-ktx:1.0.1")
    implementation("androidx.fragment:fragment-ktx:1.0.0")
    implementation("androidx.collection:collection-ktx:1.0.0")
    implementation("androidx.lifecycle:lifecycle-viewmodel-ktx:2.0.0")
```
**Espresso:** 3.2.0
**JUnit:** 4.12 
**Android Studio version:** I don't remember what version I was using then but right now I'm using 3.4.2

**Comment by mzelzoghbi** [comment]
@FrancoSabadini This issue has been fixed in the latest SDK version **8.7.0**, please feel free to reopen if the issue still persists.

Thank You
